src/ActionList/Item.tsx

```diff
@@ -178,6 +178,8 @@ const StyledItem = styled.div<
   display: flex;
   border-radius: ${get('radii.2')};
   color: ${({variant, item}) => getItemVariant(variant, item?.disabled).color};
+  // 2 frames on a 60hz monitor
+  transition: background 133.333ms linear;
```

Comment:
@dgreif that should be `33.333ms` ;)

Reply:
Great catch!  I increased it to verify I could see it locally, then forgot to remove the `1` 🤦